test/integration/integration_test.cc

```diff
@@ -947,6 +947,57 @@ TEST_P(IntegrationTest, ProcessObjectUnealthy) {
   EXPECT_THAT(response->headers(), HttpStatusIs("500"));
 }
 
+TEST_P(IntegrationTest, TrailersDroppedDuringEncoding) { testTrailers(10, 10, false, false); }
+
+TEST_P(IntegrationTest, TrailersDroppedUpstream) {
+  config_helper_.addConfigModifier(setEnableDownstreamTrailersHttp1());
+  testTrailers(10, 10, false, false);
+}
+
+TEST_P(IntegrationTest, TrailersDroppedDownstream) {
+  config_helper_.addConfigModifier(setEnableUpstreamTrailersHttp1());
+  testTrailers(10, 10, false, false);
+}
+
+TEST_P(IntegrationTest, GRPCWebTrailersNotDuplicated) {
```

Comment:
Please add a dedicated grpc_web_integration_test next to the extension tests and add this test there. In general we don't want to test non-core extensions in the main integration test code.